api/v1/endpoints/queries.sql

```diff
@@ -78,6 +78,38 @@ SELECT
 FROM
     object_data;
 
+-- BLOCK select_assessment_access_rules
+WITH object_data AS (
+    SELECT
+        a.id AS assessment_id,
+        a.tid AS assessment_name,
+        a.title as assessment_title,
+        aar.credit,
+        aar.end_date,
+        aar.id as id,
+        aar.mode,
+        aar.number,
+        aar.role,
+        aar.start_date,
+        aar.time_limit_min,
+        aar.uids
```

Comment:
Can we include all the other columns here too? https://github.com/PrairieLearn/PrairieLearn/blob/master/database/tables/assessment_access_rules.pg#L1